### ISSUE TITLE
Issue with the latest version

### ISSUE THREAD

**Issue by SneakBug8** [zioth/dokuwiki-autolink4#17]
The following is being written to dokuwiki error log regularly with the latest main branch version:

```
2023-12-17 09:17:13	/htdocs/lib/plugins/autolink4/helper.php(75)	E_WARNING: Undefined array key 1
  #0 /htdocs/lib/plugins/autolink4/helper.php(75): dokuwiki\ErrorHandler::errorHandler(2, 'Undefined array...', '/home/vol6_8/in...', 75)
  #1 /htdocs/lib/plugins/autolink4/syntax/regex.php(34): helper_plugin_autolink4->loadAndProcessConfigFile()
  #2 /htdocs/inc/Extension/PluginController.php(131): syntax_plugin_autolink4_regex->__construct()
  #3 /htdocs/inc/pluginutils.php(54): dokuwiki\Extension\PluginController->load('syntax', 'autolink4_regex', false, false)
  #4 /htdocs/inc/parser/renderer.php(117): plugin_load('syntax', 'autolink4_regex')
  #5 /htdocs/inc/parserutils.php(700): Doku_Renderer->plugin('autolink4_regex', Array, 5, 'Business Analys...')
  #6 /htdocs/inc/parserutils.php(158): p_render('xhtml', Array, NULL)
  #7 /htdocs/inc/parserutils.php(96): p_cached_output('/home/vol6_8/in...', 'xhtml', 'diary:2023:49')
  #8 /htdocs/inc/Ui/PageView.php(68): p_wiki_xhtml('diary:2023:49', 0, true, '')
  #9 /htdocs/inc/Action/Show.php(37): dokuwiki\Ui\PageView->show()
  #10 /htdocs/inc/template.php(100): dokuwiki\Action\Show->tplContent()
  #11 [internal function]: tpl_content_core('show')
  #12 /htdocs/inc/Extension/Event.php(133): call_user_func_array('tpl_content_cor...', Array)
  #13 /htdocs/inc/Extension/Event.php(199): dokuwiki\Extension\Event->trigger('tpl_content_cor...', true)
  #14 /htdocs/inc/template.php(85): dokuwiki\Extension\Event::createAndTrigger('TPL_ACT_RENDER', 'show', 'tpl_content_cor...')
  #15 /htdocs/lib/tpl/notes/main.php(56): tpl_content()
  #16 /htdocs/inc/actions.php(27): include('/home/vol6_8/in...')
  #17 /htdocs/doku.php(126): act_dispatch()
  #18 {main}
```

**Comment by Ilav1** [comment]
Some other errors are mentiened and already fixed here #15

After updating the dokuwiki files with the new code I don´t have any errors in my log.
I have only the mentiened error in the deprecated log.

**Comment by zioth** [comment]
Fixed!
